paper.md

```diff
@@ -0,0 +1,84 @@
+Some attempts have been made at creating unified packages for modern topic models, including STREAM [@stream] and TopMost [@topmost].
+These packages, however have a focus on neural models and topic model evaluation, have abstract and highly specialized interfaces, and do not include some popular topic models.
+Additionally, while model interpretation is an incredibly important aspect of topic modelling, the interpretation utilities provided in these libraries are fairly limited, especially in comparison with model-specific packages, like BERTopic.
+
+Turftopic unifies state-of-the-art contextual topic models under a superset of the `scikit-learn` [@scikit-learn] API, which users are likely already familiar with, and can be readily included in `scikit-learn` workflows and pipelines.
+We focused on making Turftopic first and foremost an easy-to-use library, that does not necessitate expert knowledge or excessive amounts of code to get started with, but gives great flexibility to power users.
+Furthermore, included an extensive suite of pretty-printing and visualization utilities that aid users in interpreting their results.
+The library also includes three topic models, which to our knowledge only have implementations in Turftopic, these are: KeyNMF [@keynmf], S^3^ [@s3], and GMM.
+
+# Functionality
+
+Turftopic includes a wide array of contextual topic models from the literature, these include:
+FASTopic [@fastopic], Clustering models, such as BERTopic [@bertopic_paper] and Top2Vec [@top2vec], auto-encoding topic models, like CombinedTM [@ctm] and ZeroShotTM [@zeroshot_tm], KeyNMF [@keynmf], Semantic Signal Separation [@s3] and GMM.
+We believe these models to be representative of the state of the art in contextual topic modelling and intend to expand on them in the future.
+
+![Components of a Topic Modelling Pipeline in Turftopic](https://x-tabdeveloping.github.io/turftopic/images/topic_modeling_pipeline.png){width="800px"}
+
+Each model in Turftopic has an *encoder* component, which is used for producing continuous document-representations, and a *vectorizer* component, which extracts term counts in each documents, thereby dictating which terms will be considered in topics.
+The user has full control over what components should be used at different stages of the topic modelling process, thereby having fine-grained influence on the nature and quality of topics.
+
+The library comes loaded with a lot of utilities to help users interpret their results, including *pretty printing* utilities for exploring topics, *interactive visualizations* partially powered by the `topicwizard` [@topicwizard] Python package, and *automated topic naming* with LLMs.
+
+To accommodate a variety of use-cases, Turftopic can be used for dynamic topic modelling, where we expect topics to change over time, can be used for uncovering hierarchical structure in topics.
```

Comment:
missing an "and it" after "change over time" I think.  also no dash in use-cases. What a "hierarchical" topic is is never explained I think - I know it's semi-intuitive, but you could simply explain it is about simultaneous extraction of topics and multiple levels of granularity?